sdk/storage/storage-internal-avro/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@azure/storage-internal-avro",
```

Comment:
does this package actually get used anywhere?

Reply:
yes, by blob https://github.com/Azure/azure-sdk-for-js/blob/7f96a18c18bb6b97ff005d54c663b56d710ff449/sdk/storage/storage-blob/src/utils/BlobQuickQueryStream.ts#L7

Reply:
does it need to be its own package or can it be absorbed into blob?

Reply:
currently it is also used by @azure/storage-blob-changefeed which is only in preview.  We could make copies of code in blob for now.  @EmmaZhu any thoughts?

Reply:
Reverted adding `@azure/storage-internal-avro`